jupyter_notebooks/exercises/GitPush_exercise3.ipynb

```diff
@@ -353,6 +353,8 @@
    "source": []
```

Comment:
<p>accidentally removed the newline, sorry</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eigen-carmona/mlreview_notebooks/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>solved</p><p> </p>